neuralmonkey/encoders/raw_rnn_encoder.py

```diff
@@ -11,9 +12,31 @@
 from neuralmonkey.nn.utils import dropout
 from neuralmonkey.dataset import Dataset
 
-# pylint: disable=invalid-name
-RNNCellTuple = Tuple[tf.contrib.rnn.RNNCell, tf.contrib.rnn.RNNCell]
-# pylint: enable=invalid-name
+
+# pylint: disable=too-few-public-methods
+class RNNSpec(object):
```

Comment:
Třída je to proto, že to má defaulty. NamedTuple umí defaulty až od Pythonu 3.6.